### DISCUSSION TITLE
DocumentStore.Url property not in DocumentStore object

### DISCUSSION THREAD

**Discussion by agfabregat** [ravendb/ravendb#12798] [General]
I am getting started with RavenDb in dotnet framework 4.7x c#
Installed Raven.Server and started it in 8080, fine.
Installed Raven.Client 5.2.2 from Nuget.
All the examples I find have this syntax:
var documentStore = new DocumentStore
            {
                Url = "......"
            };
But my DocumentStore object does not have a property Url, it has Urls.  I tried setting Urls with a string[1] but that doesn't work.
Any suggestions fully appreciated

**Comment by ayende** [comment]
You need to indeed set it using `new []{ "https://... "}`

Make sure that you are running the 5.2 server as well

**Comment by agfabregat** [comment]
ayende thank you so much.  I am glad you added that comment about "make sure you are running 5.2 server" because that was my problem.
Again thank you
